spec/system/admin/order_spec.rb

```diff
@@ -392,14 +392,18 @@ def new_order_with_distribution(distributor, order_cycle)
     expect(page).to have_selector '#select-customer'
 
     # And I select that customer's email address and save the order
-    select2_select customer.email, from: 'customer_search_override', search: true
+    tomselect_search_and_select customer.email, from: 'customer_search_override'
```

Comment:
I thought we could just use `select` now.

https://github.com/openfoodfoundation/openfoodnetwork/pull/8814/files#diff-5cad13c68b9482412216548c5f4976e9db9f062376532324dbe4f7790de03394

Does it not trigger tom actions? Or should we take a step back and subscribe to `onChange`?

Reply:
> I thought we could just use `select` now.

Yes, but this one is a particular case of selection: data are retrieved via request from user's input:
<img width="340" alt="Capture d’écran 2022-05-10 à 10 21 30" src="https://user-images.githubusercontent.com/296452/167583078-4774f59a-6e5d-40a0-bb8a-65f491d5ca7a.png">

That's why, in `tomselect_search_and_select` method, we emulate user input and click.